File: Source/Benchmarks/Yamo.Benchmarks/Model/Blog.vb
Namespace Model

  Public Class Blog

    Public Property Id As Int32

    Public Property Title As String

    Public Property Content As String

    Public Property Created As DateTime

    Public Property CreatedUserId As Int32

    Public Property Modified As DateTime?

    Public Property ModifiedUserId As Int32?

    Public Property Deleted As DateTime?

    Public Property DeletedUserId As Int32?

    Public Property Author As User

    Public Property Comments As List(Of Comment)

  End Class
End Namespace

File: Source/Samples/Yamo.Playground.CS/Model/Blog.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Yamo.Playground.CS.Model
{
    class Blog
    {
        public int Id { get; set; }

        public string Title { get; set; }

        public string Content { get; set; }

        public DateTime Created { get; set; }

        public int CreatedUserId { get; set; }

        public DateTime? Modified { get; set; }

        public int? ModifiedUserId { get; set; }

        public DateTime? Deleted { get; set; }

        public int? DeletedUserId { get; set; }
    }
}
